frontend/packages/dev-console/src/components/topology/shapes/WorkloadNode.tsx

```diff
@@ -120,6 +120,7 @@ const WorkloadNode: React.FC<NodeProps<WorkloadData>> = ({
                     <Status
                       title={`${build.metadata.name} ${build.status && build.status.phase}`}
                       status={build.status.phase}
+                      noTitle
                       iconOnly
```

Comment:
Since we are making a change here...

The above check:
```                {build.status.phase === 'Complete' ? (
                  <GreenCheckCircleIcon
                    alt={`${build.metadata.name} ${build.status && build.status.phase}`}
                  />
                ) : (```

is no longer necessary. The Status component now uses the same icon for `Complete`

Reply:
yes @jeff-phillips-18 makes sense